src/main/resources/log4j2.properties

```diff
@@ -19,7 +19,7 @@ io.strimzi.kafka.bridge.appenderRef.console.ref = STDOUT
 io.strimzi.kafka.bridge.appenderRef.additivity = false
 
 # HTTP OpenAPI specific logging levels (default is INFO)
```

Comment:
```suggestion
# HTTP OpenAPI specific logging levels (default is INFO)
# Logging healthy and ready endpoints is very verbose because of Kubernetes health checking.
```